### PR TITLE
Adjust to move to GitHub

### PR DESCRIPTION
**Pull request by eemeli**
- Update contact details, including new repo URL
- Drop mercurial & arcanist configs
- Add `.gitignore`
- Use GitHub Actions instead of Travis CI

### CODE REVIEW

**Comment by eemeli** [comment]
The CI tests don't seem to do anything under Python 2.7 & fail in Windows. Do we actually care about either of those, or could we just drop them?

**Comment by eemeli** [comment]
Actually, I know that we don't care about Python 2, so let's just drop that.

File: .arcconfig
Comment (flodolo) [inline .arcconfig:1]: Do we need to communicate something about this to the Phabricator team?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: ["3.7"]
```
Comment (flodolo) [inline .github/workflows/ci.yaml:16]: Should we expand this matrix? For comparison, compare-locales goes from 3.7 to 3.10

File: README.md [hunk 2]
```diff
@@ -36,9 +36,3 @@ The migrations are run as python modules, so you need to have their file locatio
 An example would look like
 
     $ migrate-l10n --lang it --reference-dir gecko-strings --localization-dir l10n-central/it bug_1451992_preferences_sitedata bug_1451992_preferences_translation
-
-Contact
--------
-
- - mailing list: https://lists.mozilla.org/listinfo/tools-l10n
- - bugzilla: [Open Bugs](https://bugzilla.mozilla.org/buglist.cgi?component=Fluent%20Migration&product=Localization%20Infrastructure%20and%20Tools&bug_status=__open__) - [New Bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Localization%20Infrastructure%20and%20Tools&component=Fluent%20Migration)
```
Comment (flodolo) [inline README.md:44]: What do we want to with the Bugzilla component?

**Review by flodolo — APPROVED** [review]
Mostly questions related to the removals.

I'm OK with dropping support with Windows, although it might be worth to at least get a sense of why it's failing (and it might impact us elsewhere).

**Comment by flodolo** [comment]
One note: tried installing this repo (via `pip install -e .`) but I get `ModuleNotFoundError: No module named 'hglib'` when trying to run a migration.

Do we have an implicit or missing dependency somewhere?

File: .arcconfig
Comment (eemeli) [reply .arcconfig:1]: I've filed a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1833334) about the move.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (eemeli) [reply .github/workflows/ci.yaml:16]: Yes; I'll do that in my next PR.

File: README.md (hunk 2 above)
Comment (eemeli) [reply README.md:44]: I'd be tempted to keep it open for now, tbh. It's relatively low traffic, and the reported bugs are valid, but migrating those here would probably be a bit more work than is worthwhile.

Comment (flodolo) [reply README.md:44]: WFM. We can graveyard it later when/if we clean up the queue.

**Comment by eemeli** [comment]
> I'm OK with dropping support with Windows, although it might be worth to at least get a sense of why it's failing (and it might impact us elsewhere).

According to [the log](https://github.com/mozilla/fluent-migrate/actions/runs/4988790292/jobs/8931898070), most of the failing tests are falling over with:
```
  ...
  File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\subprocess.py", line 1148, in _execute_child
    args = list2cmdline(args)
  File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\subprocess.py", line 555, in list2cmdline
    needquote = (" " in arg) or ("\t" in arg) or not arg
TypeError: a bytes-like object is required, not 'str'
```
and then there's one (`test_wet (test_tool.TestSerialize)`) that's seeing a diff in file paths due to using `\` rather than `/` as the separator.

> One note: tried installing this repo (via `pip install -e .`) but I get `ModuleNotFoundError: No module named 'hglib'` when trying to run a migration.
> 
> Do we have an implicit or missing dependency somewhere?

As mentioned in the README, the `python-hglib` dependency is only included in the `extras_require`, so you'll need to install it locally with
```
pip install -e '.[hg]'
```

**Pull request merged**